<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPickerForm</class>
 <widget class="QWidget" name="ColorPickerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>278</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>278</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ColorPicker</string>
  </property>
  <widget class="BrightLabel" name="label_bright">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>270</width>
     <height>270</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_color">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>8</y>
     <width>32</width>
     <height>270</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_arrow">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CBrightLabel</class>
   <extends>QLabel</extends>
   <header location="global">src/brightlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
